src/containers/nav-bar.html

```diff
@@ -16,16 +16,28 @@
         </button>
         <button class="mdc-button nav-button" data-navigate-to="choose-path">
           <div class="mdc-button__ripple"></div>
-          <span class="mdc-button__label">Choose Your Path</span>
-        </button>
-        <button class="mdc-button nav-button" data-navigate-to="select-query">
-          <div class="mdc-button__ripple"></div>
-          <span class="mdc-button__label">Select Example</span>
-        </button>
-        <button class="mdc-button nav-button" data-navigate-to="tabulate-query">
-          <div class="mdc-button__ripple"></div>
-          <span class="mdc-button__label">Tabulate Example</span>
+          <span class="mdc-button__label">Show Me How</span>
         </button>
+        <div class="mdc-menu-surface--anchor">
+          <button class="mdc-button nav-menu-surface-button">
+            <div class="mdc-button__ripple"></div>
+            <span class="mdc-button__label">Examples</span>
+          </button>
+          <div class="mdc-menu-surface nav-example-menu-surface">
+            <ul class="mdc-list" role="menu" aria-hidden="true" aria-orientation="vertical" tabindex="-1">
+              <li class="mdc-list-item" role="menuitem">
+                <button class="mdc-button nav-button" data-navigate-to="select-query">
```

Comment:
While we are working on this code, could we change the references of "select-query" and "tabulate-query"? I think the IDs should be more aligned with what the section is about rather than the internal workings of it. Perhaps these two sections should be IDed as "number-of-cases-example" and "comparison-gender-and-age-example"?

Also, you will need to update the links in the readme as they rely on the IDs for page anchors.